bin/soaringspot.js

```diff
@@ -632,23 +632,28 @@ async function update_contest(contest,keys) {
         await mysql.query( escape`UPDATE competition SET lt = ${lat}, lg = ${lng},
                                                       sitename = ${location.name}`);
     }
-
+	console.log(contest.time_zone);
     const dbtz = (await mysql.query( escape`
            SELECT tz, LEFT((TIMEDIFF(CONVERT_TZ(NOW(),'+00:00',${contest.time_zone}),NOW())),6) newtz
              FROM competition`))[0];
 
-    if( dbtz ) {
-        // Extract timezone
-        // probably wrong for tz 00:00, think it's in the ocean
-        let newtz = dbtz.newtz.replace(/:$/,'')
-        if( !newtz.match(/^[+-]/)) {
-            newtz = "+"+newtz;
-        }
-        if( newtz != dbtz.tz ) {
-            console.log( "current tz: "+dbtz.tz+" changing to "+newtz);
-            console.log( await mysql.query( escape`UPDATE competition set tz=${newtz}, tzoffset=time_to_sec(TIMEDIFF(CONVERT_TZ(NOW(),'+00:00',${newtz}),NOW())) `) );
-        }
-    }
+	if( dbtz.newtz)	{
```

Comment:
Good catch. I think 
`if (dbtz && dbtz.newtz) {`
would be a better approach, as this will fail hard if there are no rows in the competition database. Could you also please keep the whitespace the same as most of these lines are only whitespace changes!